blogposts/2021/dev-tool-time-adam-gordon-bell.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Writing prose like code: Dev Tool Time with Adam Gordon Bell'
+description: 'Adam Gordon Bell, Developer Advocate at Earthly, shares how he leverages dev tools to write consistent, high-quality technical prose.'
+author: Scott Bailey
+authorUrl: https://about.sourcegraph.com/handbook/company/team#scott-bailey-he-him
+publishDate: 2021-10-06T10:00-07:00
+tags: [blog]
+slug: dev-tool-time-adam-gordon-bell
+heroImage: https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/linkedin.jpg
+socialImage: https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/linkedin.jpg
+published: true
+---
+
+As a developer advocate, [Adam Gordon Bell](https://twitter.com/adamgordonbell) writes prose as a central part of his work. As a programmer, he wants the same experience writing prose that he has writing code, where linters and build pipelines help with consistency, formatting, and error checking. When Adam joined us for [Dev Tool Time](https://info.sourcegraph.com/dev-tool-time) on September 29, 2021, he shared his favorite dev tools for writing and how Earthly helps keep his published writing high quality.
+
+You can watch the recording of this episode at the [bottom of this blog post](#Watch-the-recording), but first let’s check out Adam’s desk setup:
+
+## Hardware
+
+Adam drives two monitors with a 16" Macbook Pro hooked into a CalDigit Thunderbolt dock. He uses a Logitech gaming mouse, whose thumb buttons ease browser navigation.
+
+![Adam’s current desk setup](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/current-computer.jpeg)
+
+Notably, Adam uses a 45lb iron weight, directly under his Macbook, as a heatsink. Impressively, his sit/stand barely notices the weight, even when it’s joined by Adam’s one-eyed shelter cat, Griswald.
+
+![Adam’s 45lb weight heatsink](https://sourcegraphstatic.com/blog/dev-tool-time/adam-gordon-bell/heatsink.jpeg)
```

Comment:
I think we could cut this pic – while we don't have much flexibility in formatting images it's showing up very big. Maybe you could add a caption to the image above that draws attention to Griswald instead?